packages/charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -259,15 +267,17 @@ export class LineChartBase extends React.Component<
     for (let i = 0; i < this._points.length; i++) {
       const legendVal: string = this._points[i].legend;
       const opacity: number = this.state.activeLegend === legendVal || this.state.activeLegend === '' ? 1 : 0.1;
+      const isHoverShow: boolean =
+        this.state.activeLegend === legendVal || this.state.activeLegend === '' ? true : false;
```

Comment:
No need for a ternary operator here, this can just be `this.state.activeLegend === legendVal || this.state.activeLegend === ''`. Also `isHoverShow` could then be used to determine the value of `opacity` rather than having to do the comparison of `this.state.activeLegend === legendVal || this.state.activeLegend === ''` twice.